v2/azure-arm.yaml

```diff
@@ -542,6 +542,22 @@ status:
 #     implemented as an extension on the resource type in question.
 #     Only valid for resource types.
 #
+# $azureGeneratedConfigs: <map of string -> string>
+#     The key of the map is the name of the property to export, the value of the map is a json-path like expression
+#     to the property to export. Collections (or really any fancy jsonpath expressions) are not currently supported.
+#     Only valid for resources.
+#     Example:
+#         BlobEndpoint: .Status.PrimaryEndpoints.Blob
+#
+# $manualAzureGeneratedConfigs: <array of strings>
+#     An array of names identifying the properties which will be exported to config maps manually. Prefer $azureGeneratedConfigs
+#     if the property is on Status or otherwise accessible on the resource itself. $manualAzureGeneratedConfigs can be used
+#     to have more control over exactly what value goes into the exported config map, at the cost of needing to manually implement
+#     the config map export The key of the map is the name of the property to export, the value of the map is a json-path like expression
+#     to the property to export. Collections (or really any fancy jsonpath expressions) are not currently supported.
+#     Example:
```

Comment:
Do we need a better example here? This one looks identical to `$azureGeneratedConfigs` example which does not really explain the difference in both.

Reply:
Fixed, they actually weren't supposed to have the same example.

Reply:
Fixed, I think